R/derive_var_dthcaus.R

```diff
@@ -219,6 +231,21 @@ derive_var_dthcaus <- function(dataset,
 #'
 #' @param date A character vector to be used for sorting `dataset`.
 #'
+#' @param date_imputation The value to impute the day/month when a datepart is missing.
+#'
+#'   If `NULL`: no date imputation is performed and partial dates are returned as missing.
+#'
+#'   Otherwise, a character value is expected, either as a
+#'   - format with day and month specified as 'mm-dd': e.g. '06-15' for the 15th
+#'   of June
+#'   - or as a keyword: 'FIRST', 'MID', 'LAST' to impute to the first/mid/last day/month.
+#'
+#'   Default is `NULL`
+#'
+#' @param order Sort order
+#'
+#'   Additional character vector to be used for sorting `dataset` to avoid duplicate record warning.
+#'
 #' @param mode One of `"first"` or `"last"`.
 #' Either the `"first"` or `"last"` observation is preserved from the `dataset`
 #' which is ordered by `date`.
```

Comment:
This needs to be updated, e.g., "which is ordered by the (imputed) `date` and `order`."

Reply:
updated as suggested